pandora_server/DEBIAN/prerm

```diff
@@ -60,10 +60,14 @@ get_distro () {
 GET_DISTRO="`get_distro`"
 DISTRO=`echo $GET_DISTRO | cut -f 1 -d ":"`
 
-rm /etc/init.d/pandora_server
-rm /etc/init.d/tentacle_serverd
-update-rc.d pandora_server remove
-update-rc.d tentacle_serverd remove
+if [ -x `command -v systemctl` ]; then
+  systemctl disable pandora_server
+  systemctl disable tentacle_serverd
+  systemctl daemon-reload
+else
```

Comment:
I don't know of any package from Debian that remove its init.d scripts in prerm.

Also it seems that update-rc.d works in a different way in a system with dependency boot enabled (sysvinit >= 2.88dsf-28). [1]

[1] https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=680293